src/Microsoft.AspNetCore.Mvc.RazorPages/DependencyInjection/RazorPagesOptionsExtensions.cs

```diff
@@ -61,7 +61,30 @@ public static RazorPagesOptions AuthorizePage(this RazorPagesOptions options, st
         }
 
         /// <summary>
-        /// Adds a <see cref="AuthorizeFilter"/> with the specified policy to all page under the specified path.
+        /// Adds a <see cref="AuthorizeFilter"/> to the page with the specified path.
+        /// </summary>
+        /// <param name="options">The <see cref="RazorPagesOptions"/> to configure.</param>
+        /// <param name="path">The path of the Razor Page.</param>
+        /// <returns>The <see cref="RazorPagesOptions"/>.</returns>
+        public static RazorPagesOptions AuthorizePage(this RazorPagesOptions options, string path)
+        {
+            if (options == null)
+            {
+                throw new ArgumentNullException(nameof(options));
+            }
+
+            if (string.IsNullOrEmpty(path))
+            {
+                throw new ArgumentException(Resources.ArgumentCannotBeNullOrEmpty, nameof(path));
+            }
+
+            var authorizeFilter = new AuthorizeFilter(string.Empty);
```

Comment:
Instead of doing this, just call the other overload: 
`public static RazorPagesOptions AuthorizePage(this RazorPagesOptions options, string path) => AuthorizePage(options, path, policy: string.Empty);`